swarm/network/kademlia.go

```diff
@@ -358,6 +358,10 @@ func (k *Kademlia) Off(p *Peer) {
 	}
 }
 
+//EachBin iterates over each connection of a kademlia table, per bin.
+//So `po` will be representing the bin value (0..255), while `val` represents an
+//existing connection inside that bin.
+//For any such connection found, the parameter func `eachBinFunc` will be executed
```

Comment:
I've also made a comment for this. Which one do we keep?

https://github.com/ethersphere/go-ethereum/pull/1069/files#diff-edfb4786e058d49a6335560cb9f86d00R362